data/json/mapgen/drug_lab.json

```diff
@@ -0,0 +1,50 @@
+[
+  {
+    "method": "json",
+    "om_terrain": "drug_lab",
+    "type": "mapgen",
+    "weight": 100,
```

Comment:
Since at the moment there's only one variant of this location, this field effectively ain't doing anything useful right now. Not technically an error, just a nitpick.